pkg/ingester/pyroscope/ingest_adapter.go

```diff
@@ -164,6 +167,13 @@ func (p *pyroscopeIngesterAdapter) parseToPprof(ctx context.Context, in *ingesti
 	if err != nil {
 		return fmt.Errorf("parsing IngestInput-pprof failed %w", err)
 	}
+	if len(plainReq.Series) == 0 {
+		tenantID, _ := tenant.ExtractTenantIDFromContext(ctx)
```

Comment:
do not try to push pprofs to distributor if there no pprofs (for exampel misconfigured jfr with zero samples)